specs/merge/fork-choice.md

```diff
@@ -66,6 +66,10 @@ def notify_forkchoice_updated(self: ExecutionEngine,
 *Note*: The call of the `notify_forkchoice_updated` function maps on the `POS_FORKCHOICE_UPDATED` event defined in the [EIP-3675](https://eips.ethereum.org/EIPS/eip-3675#definitions).
 As per EIP-3675, before a post-transition block is finalized, `notify_forkchoice_updated` must be called with `finalized_block_hash = Hash32()`.
 
+*Note*: Client software must not call this function as long as the paylod of the head of the chain is empty, i.e. `get_head(store).body.payload == ExecutionPayload()`.
```

Comment:
This statement is too strong. The validator that builds the merge transition block will call `notify_forkchoice_updated` when their store has an empty payload to kick off the build process

Reply:
Instead, it might be. "must not call this function as long as the transition conditions are not met on the PoW chain. i.e. there exists a block for which `is_valid_terminal_pow_block` returns true"

Reply:
btw maybe use capital "MUST NOT".

Reply:
Should be fixed now.